Buildings/Experimental/DHC/CentralPlants/Cooling/Subsystems/CoolingTowerWithBypass.mo

```diff
@@ -0,0 +1,391 @@
+within Buildings.Experimental.DHC.CentralPlants.Cooling.Subsystems;
+model CoolingTowerWithBypass
```

Comment:
```suggestion
model CoolingTowersWithBypass
```

Reply:
Thank you for your thoughtful suggestion, we have updated the model’s name.